cernopendata/templates/cernopendata_records_ui/records/record_detail.html

```diff
@@ -336,6 +336,12 @@ <h2>How were these data validated?</h2>
 
     {% if record.usage %}
     <div>
+        {% if record.collections and "OPERA-Detector-Events" in record.collections %}
+        <h2>Visualize OPERA Event</h2>
```

Comment:
"Visualise OPERA event", to use British English as elsewhere.  ("-ise" vs "-ize", and lowercase additional words in the title)